code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -1383,6 +1384,7 @@ var/list/cyborg_list = list()
 	lawsync()
 	undeployment_action.Grant(src)
 	destroy_action.Grant(src)
+	rbPDA.loc = src		//move the AI PDA to the robot
```

Comment:
unforeseen consequences

Reply:
I changed it to use forceMove instead of setting the loc, if the PDA isn't moved to the robot then they wont get any of their PDA verbs